Mathlib/FieldTheory/KrullTopology.lean

```diff
@@ -178,6 +178,17 @@ instance krullTopology (K L : Type*) [Field K] [Field L] [Algebra K L] :
 instance (K L : Type*) [Field K] [Field L] [Algebra K L] : TopologicalGroup (L ≃ₐ[K] L) :=
   GroupFilterBasis.isTopologicalGroup (galGroupBasis K L)
 
+open scoped Topology in
+lemma krullTopology_mem_nhds_one (K L : Type*) [Field K] [Field L] [Algebra K L]
```

Comment:
It seems to me that the topology is not called "Krull topology" in Mathlib, am I wrong? If this is the case, this lemma shuold not refer to it. 

BTW Why don't you state it for subgroups rather than subsets? And do you have a corresponding statement for closed ones, removing the finiteness assumption/conclusion?

Reply:
I think in Mathlib it is called Krulltopology rather than induced from the open compact topology, I think it is the only topology we have on Galois group for now. 
And can you explain a bit more about the second problem? I didn't get it clearly.

Reply:
Oh, I see! The problem is that the file `KrullTopology` is not in the Galois subfolder, this seems odd. After this PR will be closed, can you please open a discussion on Zulip to see if we should not move it there?

Concerning the second point, I was wondering if rather than assuming that `s` is a set belonging to the neighbourhoods of `1` you cannot use the `OpenSubgroup` structure; for the "closed" point, what I was asking is if you can prove (or if you can point to me at where this is in the library) that every subgroup fixing a field is closed (here you prove that every subgroup fixing a finite extension is open, so in particularly closed).

Reply:
Including fixing subgroup is closed, fixing subgroup normal iff Galois and fixing subgroup open iff finite dimensional are all proved in #16982, this lemma exist is initially because it is very frequently used in #16982.

Reply:
OK, I see, no problem. On the other hand, why do you prefer that `S` be a set rather than a term of `OpenSubgroup`?

Reply:
This commit below is about this, I am pretty sure I have pushed.
![图片](https://github.com/user-attachments/assets/759e518a-ef72-484c-ae92-43f236b561ce)

Reply:
Because when proving, a set is  taken as complement of a subgroup that is trying to be prove closed, using continuousMul to move it to the neighborhood of `1` then pick a `OpenSubgroup` inside.